subgraphs/uniswap-forks/protocols/sushiswap/src/common/handlers/handleRewardMini.ts

```diff
@@ -0,0 +1,91 @@
+import { BigDecimal, BigInt, ethereum, log } from "@graphprotocol/graph-ts";
+import { NetworkConfigs } from "../../../../../configurations/configure";
+import { MiniChefSushiswap } from "../../../../../generated/MiniChef/MiniChefSushiswap";
+import { LiquidityPool, _HelperStore } from "../../../../../generated/schema";
+import { BIGINT_FIVE, BIGINT_ZERO, INT_ZERO, UsageType, ZERO_ADDRESS } from "../../../../../src/common/constants";
+import { getOrCreateToken } from "../../../../../src/common/getters";
+import { findNativeTokenPerToken, updateNativeTokenPriceInUSD } from "../../../../../src/price/price";
+import { getRewardsPerDay } from "../../../../../src/common/rewards";
+
+export function handleRewardMini(event: ethereum.Event, pid: BigInt, amount: BigInt, usageType: string): void {
+  let masterChefPool = _HelperStore.load(pid.toString());
+  let poolContract = MiniChefSushiswap.bind(event.address);
+
+  // Create entity to track masterchef pool mappings
+  if (!masterChefPool) {
+    masterChefPool = new _HelperStore(pid.toString());
+    let getlpAddress = poolContract.try_lpToken(pid);
+    let lpTokenAddress = ZERO_ADDRESS;
+    if (!getlpAddress.reverted) {
+      lpTokenAddress = getlpAddress.value.toHexString();
+    }
+    masterChefPool.valueString = lpTokenAddress;
+    masterChefPool.valueBigInt = BIGINT_ZERO;
+    masterChefPool.save();
+  }
+
+  // Return if pool does not exist - Banana tokens?
+  let pool = LiquidityPool.load(masterChefPool.valueString!);
+  if (!pool) {
+    return;
+  }
+
+  // Update staked amounts
+  if (usageType == UsageType.DEPOSIT) {
+    pool.stakedOutputTokenAmount = pool.stakedOutputTokenAmount!.plus(amount);
+  } else {
+    pool.stakedOutputTokenAmount = pool.stakedOutputTokenAmount!.minus(amount);
+  }
+
+  // Return if you have calculated rewards recently
+  if (event.block.number.minus(masterChefPool.valueBigInt!).lt(BIGINT_FIVE)) {
```

Comment:
maybe name the constant RECENT_BLOCK_RANGE or RECENT_BLOCK_THRESHOLD?

Reply:
Renamed